api/auth.js

```diff
@@ -8,116 +8,116 @@ const JWT_SECRET = process.env.JWT_SECRET;
 
```

Comment:
Ok I dont know why this shows a lot of changes, maybe it was cause of prettier but the only changes were that I added tot he response the users id. Everything is still working the same.